dependencies.yaml

```diff
@@ -477,34 +466,15 @@ dependencies:
               - *pytorch_pip
```

Comment:
Note: it looks like we're using the same `*pytorch_pip` and `*tensordict` package outputs for `[requirements, pyproject]` for CUDA 11, CUDA 12, and empty CUDA matrices. We could move this to `common:` instead of `specific:`.

Reply:
done